src/useSpeechRecognition.js

```diff
@@ -26,11 +26,18 @@ const useSpeechRecognition = (props = {}) => {
 
   const listen = (args = {}) => {
     if (listening) return;
-    const { lang = '', interimResults = true } = args;
+    const {
+      lang = '', interimResults = true, continuous = false, maxAlternatives = 1, grammars
+    } = args;
     setListening(true);
     recognition.current.lang = lang;
     recognition.current.interimResults = interimResults;
     recognition.current.onresult = processResult;
+    recognition.current.continuous = continuous;
+    recognition.current.maxAlternatives = maxAlternatives;
+    if (grammars) {
+      recognition.current.onresult = grammars;
```

Comment:
should this be
```js
recognition.current.grammars = grammars;
```

Reply:
@MikeyParton check it again in new revision. Fixed that line.